src/tests/test_network.cpp

```diff
@@ -119,8 +119,9 @@ TEST_F(TestNetwork, ResourceManager)
     EXPECT_TRUE(d_ucx_block.remote_handle());
     EXPECT_TRUE(d_ucx_block.remote_handle_size());
 
-    // this is generally true, but perhaps we should not count on it
-    EXPECT_LE(h_ucx_block.remote_handle_size(), d_ucx_block.remote_handle_size());
+    // the following can not assumed to be true
+    // the remote handle size is proportional to the number and types of ucx transports available in a given domain
+    // EXPECT_LE(h_ucx_block.remote_handle_size(), d_ucx_block.remote_handle_size());
```

Comment:
Would it be better to put this comment in the `remote_handle_size()` function documentation?

Reply:
Not really, users will never access it directly and it's not exposed publicly. 

The buffer that holds the remote keys will only get packed opaquely has part of a remote descriptor

Reply:
I can add a component there as well. But really I was just making a silly assumption in the tests based on a data point of N=1.

Reply:
Fair enough